### ISSUE TITLE
Dashboard: add chart with number of posts each week

### ISSUE THREAD

**Issue by tdziurko** [jvm-bloggers/jvm-bloggers#78]
Replace example counters with a real logic and real values.

UI view: http://screencast.com/t/EC8nVxX9ay

**Comment by mszarlinski** [comment]
I can do this after #121

**Comment by tdziurko** [comment]
This is not so straightforward as it might seem. I will create a discussion about this as a separate issue.

**Comment by tdziurko** [comment]
Not needed at this moment.
